lib/verifier.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "journal"
+require "services"
+
+# Common superclass for all things Verifier.
+
+module PostZephirProcessing
+  class Verifier
+    attr_reader :errors
+
+    def initialize
+      # Mainly for testing
+      @errors = []
+    end
+
+    # Verify all dates listed in the journal
+    def run
+      Journal.from_yaml.dates.each do |date|
+        run_for_date(date: date)
+      end
+    end
+
+    # Verify outputs for one date.
+    # Useful for verifying datestamped files.
+    def run_for_date(date:)
```

Comment:
We should log at an `INFO` level when a verifier starts, and make sure subclasses call `super` for `run_for_date`